backend/pick-git/build.gradle

```
@@ -17,6 +17,7 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-validation'
+	implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5'
```

```suggestion
	implementation 'org.apache.httpcomponents:httpclient:4.5'
```

이렇게 형식 통일하는 거 어떤가요?!